web/frontend/src/components/camera/stream-manager.ts

```diff
@@ -0,0 +1,33 @@
+import type {
+  Client,
+} from '@viamrobotics/sdk';
+import { CameraManager } from './camera-manager';
+
+export class StreamManager {
+  isConnected: boolean;
```

Comment:
What is the purpose of this if it's always true?

Reply:
Opps logic from a forgotten idea. It has been scrubbed